config/eslint-config/README.md

```diff
@@ -10,42 +10,55 @@ npm install -D @ilo-org/eslint-config
 
 ## Usage
 
-You can use `@ilo-org/eslint-config` in your project by extending it in your
-`eslint` configuration. For example, if we had an `.eslintrc` file:
+`@ilo-org/eslint-config` provides a list of ESLint flat configurations:
 
-```json
-{
-  "extends": ["@ilo-org/eslint-config"]
-}
+| Config Name | Description                           | Internal Extends |
+| ----------- | ------------------------------------- | ---------------- |
+| js          | Base JavaScript config                | ⛔               |
+| recommended | Recommended configuration `js` + `ts` | `js`             |
+| react       | React Typescript config               | `ts` -> `js`     |
+
+You can use any configuration you like by simply including it inside your eslint config
+
+Just export it
```

Comment:
"Just export it" sounds a bit confusing since the config has to be imported first. I think it would be better if you say that it can be re-exported from the project level config.

Also, can't we do this?

```
export { recommended as default } from "@ilo-org/eslint-config";
```

Reply:
No, we don't have a named export.